platform/android/export/export_plugin.cpp

```diff
@@ -1005,6 +1005,57 @@ void EditorExportPlatformAndroid::_write_tmp_manifest(const Ref<EditorExportPres
 	store_string_at_path(manifest_path, manifest_text);
 }
 
+void EditorExportPlatformAndroid::_fix_themes_xml(const Ref<EditorExportPreset> &p_preset) {
+	const String themes_xml_path = ExportTemplateManager::get_android_build_directory(p_preset).path_join("res/values/themes.xml");
+	bool enable_swipe_to_dismiss = p_preset->get("wear_os/swipe_to_dismiss");
+
+	if (!FileAccess::exists(themes_xml_path)) {
+		print_error("res/values/themes.xml does not exist.");
+		return;
+	}
+
+	String xml_content;
+	Ref<FileAccess> file = FileAccess::open(themes_xml_path, FileAccess::READ);
+	PackedStringArray lines = file->get_as_text().split("\n");
+	file->close();
+
+	// Check if the line contains the existing <item name="android:windowSwipeToDismiss"> element.
+	// If found and `enable_swipe_to_dismiss` is true, remove the item.
+	// If found and `enable_swipe_to_dismiss` is false, update its value to "false".
```

Comment:
@syntaxerror247 FYI, the comments were not updated to reflect the new logic.

Reply:
@m4gr3d, thanks for catching that! The comments are definitely out of date. I'm actually planning to refactor the variable names in this file too, so I'll update the comments and make those changes together.